composer.json

```diff
@@ -53,8 +53,11 @@
     "conflict": {
         "doctrine/doctrine-bundle": "<2.7",
         "gedmo/doctrine-extensions": "<3.7",
-        "symfony/framework-bundle": "<5.4"
+        "symfony/framework-bundle": "<5.4",
+        "symfony/string": "5.4.0-BETA1 || 5.4.0-RC1"
```

Comment:
This is how I solved it at the end @VincentLanglet . Composer for some reason prefers to install this rc and beta and keep translation contracts up to v3 instead of lower it down to v2 and use latest 5.4 of symfony string. Seems like a bug in composer but not sure when I will have time for it.

Reply:
Maybe worth to open an issue ?